src/core/server/http/index.ts

```diff
@@ -19,20 +19,26 @@
 
 import { Observable } from 'rxjs';
 
-import { Env } from '../config';
 import { LoggerFactory } from '../logging';
 import { HttpConfig } from './http_config';
 import { HttpService } from './http_service';
+import { Router } from './router';
 
 export { Router, KibanaRequest } from './router';
 export { HttpService };
+export { HttpServerInfo } from './http_server';
+export { BasePathProxyServer } from './base_path_proxy_server';
 
 export { HttpConfig };
 
 export class HttpModule {
   public readonly service: HttpService;
 
-  constructor(readonly config$: Observable<HttpConfig>, logger: LoggerFactory, env: Env) {
-    this.service = new HttpService(this.config$, logger, env);
+  constructor(readonly config$: Observable<HttpConfig>, logger: LoggerFactory) {
+    this.service = new HttpService(this.config$, logger);
+
+    const router = new Router('/core');
+    router.get({ path: '/', validate: false }, async (req, res) => res.ok({ version: '0.0.1' }));
```

Comment:
> Hmm, is this supposed to be here?

We had this in master already, I just moved it here: https://github.com/elastic/kibana/blob/6034cc7a6323653857f15bb27b8de9bbf192a6e7/src/core/server/index.ts#L38-L40

> Are we going to start defining a version for core?

Nope, the idea was just to have one random new platform/core endpoint so that we can do a quick manual smoke test when we touch core HTTP logic. It was supposed to be replaced by a proper unit/integration test for the first _real_ core endpoint once we have it.

I don't have a strong opinion on that, do you think we should get rid of it?

Reply:
Okay, I guess if it's not new then it's less of a deal. I do think we should get rid of it but I don't think it needs to happen in this PR.

Reply:
Okay, will get rid of it then, altogether with any other changes that will be needed based on your review :)